pages/index.tsx

```diff
@@ -10,11 +11,12 @@ export default function Home() {
       </Head>
 
       <main>
-        <Logo></Logo>
-        <div></div>
+        <Link href={`/traveling/`}>
+          <a>
+            <Logo></Logo>
+          </a>
+        </Link>
       </main>
-
-      <footer></footer>
     </div>
```

Comment:
Es scheint so, als ob dieses <div> lediglich dazu da ist, dass React das als eine Komponente rendern kann. Es hat jedoch semantisch keinen Nutzen. Hierfür kannst du einfach [Fragments](https://reactjs.org/docs/fragments.html#short-syntax) nutzen und auf das unnötig div verzichten
```suggestion
    <>
```